corehq/apps/userreports/reports/view.py

```diff
@@ -573,9 +573,9 @@ def export_table(self):
     def email_response(self):
         with closing(StringIO()) as temp:
             export_from_tables(self.export_table, temp, Format.HTML)
-            return self.render_json_response({
+            return HttpResponse(json.dumps({
                 'report': temp.getvalue(),
-            })
+            }), content_type='application/json')
```

Comment:
For my understanding, how does this solve the underlying issue?

Reply:
good question. `render_json_response` calls `json.dumps` with [a few extra arguments](https://github.com/brack3t/django-braces/blob/master/braces/views/_ajax.py#L41-L44) including [`ensure_ascii=False`](https://github.com/brack3t/django-braces/blob/master/braces/views/_ajax.py#L33) and then reencodes it. 

This apparently chokes if the data is previously encoded ([which it is](https://github.com/dimagi/commcare-hq/pull/18022/files#diff-5fe4ce2e18624b1a325fb22b1b0a01b7R264)). 

Just some obscure coincidence that this broke us basically, which was why I wrote the high-level test.

Reply:
Thanks.